app/views/finance/lead_providers/show.html.erb

```diff
@@ -0,0 +1,98 @@
+<% content_for :title, "Lead Provider Payment Breakdown" %>
+<%
+  def band_index_to_heading(number)
+    ("A".."Z").to_a[number]
+  end
+%>
+
+<h1 class="govuk-heading-l"><%= @ecf_lead_provider.name %></h1>
+
+<table class="govuk-table">
+  <caption class="govuk-table__caption govuk-table__caption--l">ECF Payment breakdown</caption>
+  <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header govuk-!-width-one-half">Milestone</th>
+        <td class="govuk-table__cell" data-test="lead-provider-name">Started</td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header">Recruitment target</th>
+        <td class="govuk-table__cell" data-test="lead-provider-name"><%= @ecf_lead_provider.call_off_contract[:recruitment_target] %></td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header">Current ECTs</th>
+        <td class="govuk-table__cell" data-test="lead-provider-name"><%= @total_ect %></td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header">Current mentors</th>
+        <td class="govuk-table__cell" data-test="lead-provider-name"><%= @total_mentors %></td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header">Current participants</th>
+        <td class="govuk-table__cell" data-test="lead-provider-name"><%= @total_participants %></td>
+      </tr>
+  </tbody>
+</table>
+
+<table class="govuk-table">
+  <caption class="govuk-table__caption govuk-table__caption--l">Service Fee</caption>
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th class="govuk-table__header">Band</th>
+      <th class="govuk-table__header">Number of Participants</th>
+      <th class="govuk-table__header">Payment amount per person</th>
+      <th class="govuk-table__header">Payment amount monthly</th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+    <% @breakdown[:service_fees].each_with_index do |service_fee, i| %>
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell"><%= band_index_to_heading(i) %></td>
```

Comment:
Or maybe the value of `band_index_to_heading` should be in the `service_fee` - I imagine wherever we want to display them it would be useful to know whether it's band A, B or C.

Reply:
Another candidate for decorators